contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -444,6 +449,16 @@ contract TrueCreditAgency is UpgradeableClaimable, ITrueCreditAgency {
         emit EnteredDefault(borrower, reason);
     }
 
+    function pokeBorrowLimitTimer(ITrueFiPool2 pool, address borrower) public {
+        if (borrowLimit(pool, borrower) > 0) {
+            overBorrowLimitTime[pool][borrower] = 0;
+            return;
+        }
+        if (overBorrowLimitTime[pool][borrower] == 0) {
+            overBorrowLimitTime[pool][borrower] = block.timestamp;
+        }
```

Comment:
Won't this also be triggered when the borrowed amount is strictly equal to max borrow limit?